### PR TITLE
Feature/db scripts

### PR DESCRIPTION
**Pull request by PhumzileNkosi**

### CODE REVIEW

**Review by nikitaabrey — APPROVED** [review]

**Review by thaboRach — COMMENTED** [review]
This looks like SQLServer syntax, I thought we decided to MySQL

File: database/DBCreation.sql [hunk 1]
```diff
@@ -0,0 +1,60 @@
+
+CREATE TABLE  Difficulty (
+	DifficultyID int NOT NULL,
+	DifficultyLevel varchar(250) NOT NULL,
+	PRIMARY KEY(DifficultyID)
+);
+CREATE TABLE Recipe (
+	RecipeID INT IDENTITY(1,1) PRIMARY KEY,
```
Comment (thaboRach) [inline database/DBCreation.sql]: Not MySQL syntax,
```suggestion
	RecipeID int NOT NULL,
```

File: database/DBCreation.sql [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+	Name VARCHAR(150) NOT NULL,
+	Description VARCHAR(255),
+	Instructions VARCHAR(255),
+	PrepTime FLOAT,
+	Difficulty INT,
+	FOREIGN KEY (Difficulty) REFERENCES Difficulty(DifficultyID)
+);
+
+CREATE TABLE Coffee (
+  CoffeeID int  IDENTITY(1,1)  NOT NULL,
```
Comment (thaboRach) [inline database/DBCreation.sql]: Not MySQL syntax, try
```suggestion
  CoffeeID int  NOT NULL,
```

File: database/DBCreation.sql [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+CREATE TABLE Unit (
+	UnitID int NOT NULL,
+	UnitName int NOT NULL, 
+	PRIMARY KEY(UnitID)
+);
+
+
+CREATE TABLE Ingredient (
+ IngredientID INT IDENTITY(1,1) PRIMARY KEY ,
```
Comment (thaboRach) [inline database/DBCreation.sql]: Not MySQL syntax, try
```suggestion
 IngredientID int NOT NULL ,
```

**Review by thaboRach — COMMENTED** [review]
I think in general we should stick to single syntax e.g VARCHAR or varchar and INT or int

File: database/DBCreation.sql (hunk 3 above)
Comment (PhumzileNkosi) [reply database/DBCreation.sql]: @thaboRach ok, will update the script. Thanks for the feedback

File: database/DBCreation.sql [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+ Name VARCHAR(150) NOT NULL,
+);
+CREATE TABLE RecipeIngredient (
+RecipeID INT  FOREIGN KEY REFERENCES Recipe(RecipeID),
+IngredientID INT FOREIGN KEY REFERENCES Ingredient(IngredientID),
+Quantity VARCHAR(120),
+Unit INT REFERENCES Unit(UnitID),
```
Comment (thaboRach) [inline database/DBCreation.sql]: Shouldn't this be
```suggestion
UnitID int NOT NULL,
```

then later specify that it is a foriegn key
```
FOREIGN KEY (UnitID) REFERENCES Unit(UnitID)
```

**Review by thaboRach — APPROVED** [review]

**Review by Raz-BBD — APPROVED** [review]

**Pull request merged**